rust/libmagic/src/file_types.rs

```diff
@@ -0,0 +1,716 @@
+    "wbmp" => FileTypeInfo { friendly_type: "WAP Bitamp (WBMP)", mime_type: "image/vnd.wap.wbmp"},
+    "wav" => FileTypeInfo { friendly_type: "Waveform Audio File Format (WAV)", mime_type: "audio/x-wav"},
+    "davmount" => FileTypeInfo { friendly_type: "Web Distributed Authoring and Versioning", mime_type: "application/davmount+xml"},
+    "woff" => FileTypeInfo { friendly_type: "Web Open Font Format", mime_type: "application/x-font-woff"},
+    "wspolicy" => FileTypeInfo { friendly_type: "Web Services Policy", mime_type: "application/wspolicy+xml"},
+    "webp" => FileTypeInfo { friendly_type: "WebP Image", mime_type: "image/webp"},
+    "wtb" => FileTypeInfo { friendly_type: "WebTurbo", mime_type: "application/vnd.webturbo"},
+    "wgt" => FileTypeInfo { friendly_type: "Widget Packaging and XML Configuration", mime_type: "application/widget"},
+    "hlp" => FileTypeInfo { friendly_type: "WinHelp", mime_type: "application/winhlp"},
+    "wml" => FileTypeInfo { friendly_type: "Wireless Markup Language (WML)", mime_type: "text/vnd.wap.wml"},
+    "wmls" => FileTypeInfo { friendly_type: "Wireless Markup Language Script (WMLScript)", mime_type: "text/vnd.wap.wmlscript"},
+    "wmlsc" => FileTypeInfo { friendly_type: "WMLScript", mime_type: "application/vnd.wap.wmlscriptc"},
+    "wpd" => FileTypeInfo { friendly_type: "Wordperfect", mime_type: "application/vnd.wordperfect"},
+    "stf" => FileTypeInfo { friendly_type: "Worldtalk", mime_type: "application/vnd.wt.stf"},
+    "wsdl" => FileTypeInfo { friendly_type: "WSDL - Web Services Description Language", mime_type: "application/wsdl+xml"},
+    "xbm" => FileTypeInfo { friendly_type: "X BitMap", mime_type: "image/x-xbitmap"},
+    "xpm" => FileTypeInfo { friendly_type: "X PixMap", mime_type: "image/x-xpixmap"},
+    "xwd" => FileTypeInfo { friendly_type: "X Window Dump", mime_type: "image/x-xwindowdump"},
+    "der" => FileTypeInfo { friendly_type: "X.509 Certificate", mime_type: "application/x-x509-ca-cert"},
+    "fig" => FileTypeInfo { friendly_type: "Xfig", mime_type: "application/x-xfig"},
+    "xhtml" => FileTypeInfo { friendly_type: "XHTML - The Extensible HyperText Markup Language", mime_type: "application/xhtml+xml"},
+    "xml" => FileTypeInfo { friendly_type: "XML - Extensible Markup Language", mime_type: "application/xml"},
+    "xdf" => FileTypeInfo { friendly_type: "XML Configuration Access Protocol - XCAP Diff", mime_type: "application/xcap-diff+xml"},
+    "xenc" => FileTypeInfo { friendly_type: "XML Encryption Syntax and Processing", mime_type: "application/xenc+xml"},
+    "xer" => FileTypeInfo { friendly_type: "XML Patch Framework", mime_type: "application/patch-ops-error+xml"},
+    "rl" => FileTypeInfo { friendly_type: "XML Resource Lists", mime_type: "application/resource-lists+xml"},
+    "rs" => FileTypeInfo { friendly_type: "XML Resource Lists", mime_type: "application/rls-services+xml"},
```

Comment:
Not sure this one is right... Could this be an older list?

Reply:
Very possible. Also possible it's simply a file extension clash. Lots of file extensions get re-used and I think Rust probably picked one that was already in "common" use prior to its existence. :(

Reply:
I'll fix that one since I think it's infinitely more likely people will have Rust code with .rs than XML resource lists with .rs.